common/src/main/java/org/apache/uniffle/common/compression/Codec.java

```diff
@@ -55,6 +55,20 @@ public static Codec newInstance(RssConf rssConf) {
    */
   public abstract byte[] compress(byte[] src);
```

Comment:
If using the bytebuffer as the compression main object, can we remove this interface?

Reply:
It will be next pr.